doc/qbk/6.2.MutableString.qbk

```diff
@@ -0,0 +1,71 @@
+[/
+    Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:mutablestring MutableString]
+
+A ['MutableString] is a container that has the `assign` and
+`append` member functions that accept iterators to `char`.
+
+The library __decoded_view__ class provides a operations
+to assign and append decoded values to a ['MutableString].
+
+When storing decoded values in a container, these functions
+allow the caller to recycle the input string without
+reallocating memory.
+
+[heading Related Identifiers]
+
+* [link url.ref.boost__urls__decoded_view `decoded_view`]
+* [link url.ref.boost__urls__grammar__is_mutable_string `is_mutable_string`]
+
+[heading Requirements]
+
+In this table:
+
+* `T` is a type meeting the requirements of ['MutableString]
+* `s` is a mutable value of type `T`
+* `first`, `last` are iterators whose `value_type` is `char`
+
+[table Valid expressions
+[[Expression] [Type] [Semantics, Pre/Post-conditions]]
+[
+    [
+    ```
+    s.assign(first,last)
+    ```
+    ]
+    [`T&`]
+    [
+        This member function assigns the `char`s from the
+        range of characters in `[first, last)` to the
+        container `s`.
```

Comment:
You don't need to say container, you can just say `s`, because `s` was defined earlier.